src/cli-validator/utils/processConfiguration.js

```diff
@@ -363,6 +363,10 @@ const validateConfigOption = function(userOption, defaultOption) {
       optionType = option;
     }
   });
+  // return directly if there is no matched optionType
```

Comment:
Just suggesting an updated comment - I will commit it before I merge
```suggestion
  // if optionType doesn't match, there are no predefined options for this rule
```